### PR TITLE
Updating module version from 2.12.0 to 2.13.0.

### PR DESCRIPTION
**Pull request by KhoiKy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
+# 2.13.0
```
Comment (splatteredbits) [inline CHANGELOG.md:1]: 2.12.0 was never released, so add these release notes after the current 2.12.0 release notes.

File: Carbon/Carbon.psd1 [hunk 2]
```diff
@@ -25,7 +25,7 @@
     RootModule = 'Carbon.psm1'
 
     # Version number of this module.
-    ModuleVersion = '2.12.0'
+    ModuleVersion = '2.13.0'
```
Comment (splatteredbits) [inline Carbon/Carbon.psd1:28]: 2.12.0 was never released.

**Review by splatteredbits — CHANGES_REQUESTED** [review]

File: Carbon/Carbon.psd1 (hunk 2 above)
Comment (KhoiKy) [reply Carbon/Carbon.psd1]: I misread my apologies.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,15 +5,12 @@ module, now available on the PowerShell Gallery. Please switch to the new module
 functions from Carbon, a warning message will be written. These function will be removed in the next major version of
 Carbon.
 
-
-# 2.12.0 
-
 ## Added suport for PowerShell Core
```
Comment (splatteredbits) [inline CHANGELOG.md:8]: Remove this heading and all empty lines between the two "sections". Also, looks like the last item doesn't break at character/column 120. Please add if missing.

```markdown
* The `Install-CMsi`, `Get-CMsi`, and `Get-CProgramInstallInfo` functions have moved to a new `Carbon.Windows.Installer`
module, now available on the PowerShell Gallery. Please switch to the new module and update usages. If you use these
functions from Carbon, a warning message will be written. These function will be removed in the next major version of
Carbon.
* Carbon now works under PowerShell 7.
* Fixed: Get-CPermission fails to grant permissions on certificates in PowerShell 6+ and on certificates that .NET
Framework load as RSA cryptographic next generation keys, which don't have an API for setting private key permissions.
```

**Review by splatteredbits — CHANGES_REQUESTED** [review]

**Review by splatteredbits — APPROVED** [review]

**Pull request merged**